.editorconfig

```diff
@@ -13,6 +13,10 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
+[*.js]
```

Comment:
I will put aside stubbornness and play friendly with conventions since I actually have some collaboration here. 2 spaces wins. I'll probably just revert this whole file back to the original generated in 6c043c29b2f9cc41dc7d2efc75a911e405f2d4af to make any future upgrades as simple as possible.